<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzTab</class>
 <widget class="QMainWindow" name="PtzTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云台控制</string>
  </property>
  <property name="whatsThis">
   <string extracomment="0-255"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="numberOfCom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRate">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBits">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBits">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="checkBits">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="openComBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeComBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setingBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>高级设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="leftBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QPushButton" name="callPreset1Btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预置位1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openIrisBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开光圈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeIrisBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭光圈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="focusFarBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>聚焦远</string>
    </property>
   </widget>
   <widget class="QPushButton" name="focusNearBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>聚焦近</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomInBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QComboBox" name="protocolBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>云台控制协议</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像机地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="camAdressLed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
